Java/E-montazysta/src/main/java/com/emontazysta/model/dto/ToolTypeDto.java

```diff
@@ -14,12 +16,13 @@
 public class ToolTypeDto {
     private Long id;
 
+    @NotBlank(message = "Name cannot be empty")
     private String name;
-
+    @PositiveOrZero(message = "In service count number cannot be negative")
     private int inServiceCount;
-
+    @PositiveOrZero(message = "Critical number cannot be negative")
     private int criticalNumber;
-
+    @PositiveOrZero(message = "Available count number cannot be negative")
```

Comment:
```suggestion
```
This value is created on backend
https://github.com/s20146/PRO/blob/23a8b9a0ff4f1cd696ca79848103ed664bd4da22/Java/E-montazysta/src/main/java/com/emontazysta/mapper/ToolTypeMapper.java#L29